assets/check

```diff
@@ -28,6 +28,9 @@ tag_filter=$(jq -r '.source.tag_filter // ""' < $payload)
 git_config_payload=$(jq -r '.source.git_config // []' < $payload)
 ref=$(jq -r '.version.ref // ""' < $payload)
 skip_ci_disabled=$(jq -r '.source.disable_ci_skip // false' < $payload)
+filterWhitelist=$(jq -r '.source.filter.include // []' < $payload)
+filterBlacklist=$(jq -r '.source.filter.exclude // []' < $payload)
```

Comment:
nitpick: consistent variable naming style (`filter_whitelist`) - goes for all the other vars too. :sweat_smile: